contracts/src/xYield/xYieldVault.sol

```diff
@@ -157,26 +161,26 @@ contract xYieldVault is ERC4626, Ownable2Step, ReentrancyGuard, Pausable {
         return assets;
     }
 
-    // function remoteWithdraw(uint256 assets, address receiver, uint64 chainId) external nonReentrant returns (uint256)
-    // {
-    //     // TODO: Implement cross-chain withdraw coordination
-    //     revert NotImplemented();
-    // }
+    function withdrawFrom(uint256 assets, address owner, uint64 chainId) external nonReentrant returns (uint256) {
+        _withdrawFrom(owner, assets);
+    }
+
+    // NOTE - for remote withdrawals we update virtualTotalAssets after withdrawal to prevent share price inflation
+    // before
+    // global shares is updated
+    function _withdrawFrom(address owner, uint256 assets) internal {
+        // TODO - deposit into escrow instead of transferring user's underlying to this contract
```

Comment:
Aren't we supposed to bridge it to caller's chain ?

Reply:
correct. we do that via 7683, i.e. locking the user's funds in escrow until the solver fills it on the target chain + provides proof of fill